src/relay/backend/build_module.cc

```diff
@@ -524,7 +593,8 @@ class RelayBuildModule : public runtime::ModuleNode {
     }
 
     auto ext_mods = graph_codegen_->GetExternalModules();
-    ret_.mod = tvm::codegen::CreateMetadataModule(ret_.params, ret_.mod, ext_mods, GetTargetHost());
+    ret_.mod = tvm::codegen::CreateMetadataModule(ret_.params, ret_.mod, ext_mods, GetTargetHost(),
+                                                  graph_codegen_->GetAOTMetadata());
```

Comment:
It seems a bit ad-hoc to include just the AoTMetadata here. I'd suggest to include LoweredOutput here. Thus, that way inside CreateMetadataModule --> it will know how to consume the output of ExecutorCodegen to create correct metadata module. WDYT ?